jiant/proj/main/modeling/taskmodels.py

```diff
@@ -1,22 +1,22 @@
 import abc
 
-from dataclasses import dataclass
-from typing import Any
-from typing import Callable
-
 import torch
 import torch.nn as nn
 
+from typing import Callable
+
 import jiant.proj.main.modeling.heads as heads
-from jiant.proj.main.components.outputs import LogitsOutput, LogitsAndLossOutput
+
+from jiant.proj.main.components.outputs import LogitsAndLossOutput
+from jiant.proj.main.components.outputs import LogitsOutput
 from jiant.utils.python.datastructures import take_one
-from jiant.shared.model_resolution import ModelArchitectures
+
 from jiant.tasks.core import TaskTypes
 
 
 class JiantTaskModelFactory:
     """This factory is used to create task models bundling the task,
-       encoder, and task head within the task model.
+       jiant_transformers_model, and task head within the task model.
```

Comment:
Likewise, is there a reason for changing from `encoder` to `jiant_transformers_model`? (It looks like it's still using the encoder, not the whole model, so it's a little unclear to me)

Reply:
The naming change was meant to clarify that it is a `JiantTransformersModel` and not HF's `encoder`. I thought the new name would be more descriptive.